unittests/TestUtils/TestConfig.h

```diff
@@ -18,6 +18,9 @@ inline const std::string PathToTxtTestFiles(PhasarConfig::PhasarDirectory() +
 inline const std::string PathToJSONTestFiles(PhasarConfig::PhasarDirectory() +
                                              "test/json_test_code/");
 
+inline const std::string PathToSwiftTestFiles(
+    PhasarConfig::PhasarDirectory() + "build/test/llvm_swift_test_code/");
```

Comment:
I would prefer not having build directly here, because it assumes the build folder is always called like this. Which isn't the case e.g. for cmake presets, but in TestConfig.h we are assuming it already, so...

Reply:
Tbh, I just copied the existing setup for the other tests. So would it be fine for you to keep it this way? We can rework the whole setup for all tests with a different PR, this would be a bit off topic for this one I guess?

Reply:
sure